azurerm/resource_arm_postgresql_server.go

```diff
@@ -70,7 +81,18 @@ func resourceArmPostgreSQLServer() *schema.Resource {
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
 								string(postgresql.Basic),
-								string(postgresql.Standard),
+								string(postgresql.GeneralPurpose),
+								string(postgresql.MemoryOptimized),
+							}, true),
+							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
+						},
+
+						"family": {
```

Comment:
What is the relationship between this and "sku.name"? Because I found that "sku.name" also contains "Gen4" or "Gen5".